spec/models/i14y_search_spec.rb

```diff
@@ -1,11 +1,30 @@
+# frozen_string_literal: true
```

Comment:
All changes/additions replicate those added in SRCH-3614.  No net-new code here.